dhall-csv/src/Dhall/CsvToDhall.hs

```diff
@@ -1,4 +1,24 @@
-module Dhall.CsvToDhall (hello) where
+module Dhall.CsvToDhall (dhallFromCsv) where
 
-hello :: String
-hello = "Hello CSV to Dhall!"
+import Data.Text.Encoding   (decodeUtf8)
+import Data.Void            (Void)
+import Dhall.Core           (Expr, RecordField)
+import Dhall.Src            (Src)
+
+import qualified Data.Csv
+import qualified Data.HashMap.Strict    as HashMap
+import qualified Data.Sequence          as Sequence
+import qualified Dhall.Core             as Core
+import qualified Dhall.Map
+
+dhallFromCsv :: [Data.Csv.NamedRecord] -> Expr Src Void
+dhallFromCsv csv = Core.ListLit Nothing $ Sequence.fromList $ map convertRecord csv
```

Comment:
`ListLit` has an invariant that when the `Seq` is empty, the type annotation must be `(Just _)`. Can we be sure that the `Seq` will be nonempty for some reason? It might still be better style to check that `csv` is nonempty and to `error` out if it is empty.

Reply:
Yes I saw that, but I wasn't really sure of what type to use when the list was empty. Would it work with the empty record? Or would it be better to just throw an error in that case as you suggested?

Reply:
> Would it work with the empty record?

Makes sense to me! If you want to take a look at how things are represented in the AST, maybe take a look at the `dhall haskell-syntax-tree` command.

Reply:
Thanks! It was really helpful, I wasn't aware that the type should be `List {}`, I thought it was only `{}`.